dbt/include/mariadb/macros/materializations/snapshot/create_columns.sql

```diff
@@ -0,0 +1,14 @@
+{#
+    Add new columns to the table if applicable
+#}
+{% macro create_columns(relation, columns) %}
```

Comment:
This macro is not necessary, as [it already exists](https://github.com/dbt-labs/dbt-core/blob/1.7.latest/core/dbt/include/global_project/macros/materializations/snapshots/helpers.sql#L5) within the base `dbt-core` SQL templates.